### ISSUE TITLE
Background color of "graph panel" based on value

### ISSUE THREAD

**Issue by obazoud** [grafana/grafana#1365]
As an end user, I would like background change based on value like "singlestat panel".
Very useful for Operations guys.

**Comment by joune** [comment]
+1

**Comment by choucrifahed** [comment]
+1

**Comment by mzupan** [comment]
+1

**Comment by naneau** [comment]
This seems extremely useful

**Comment by utkarshcmu** [comment]
+1

**Comment by mtanda** [comment]
+1

**Comment by sckevmit** [comment]
:+1:

**Comment by LeonLee88** [comment]
+1

**Comment by majkinetor** [comment]
+1

**Comment by ibatuash** [comment]
+1

**Comment by cgimeno** [comment]
+1

**Comment by yacchin1205** [comment]
+1

**Comment by MatMeredith** [comment]
+1

**Comment by gealtrag** [comment]
+1

**Comment by infradom** [comment]
+1

**Comment by scharissis** [comment]
+1

Any progress/update on this?

**Comment by rogerlucas** [comment]
+1

**Comment by gelicia** [comment]
Does #35910 resolve this issue?

**Comment by scharissis** [comment]
@gelicia , I don't believe so, as #35910 only affects the series color. This issue pertains to the background color.

**Comment by infradom** [comment]
background color changes  (to red) are much more visible to the operations / NOC guys

**Comment by adenoz** [comment]
To be able to change the background color at all, natively without plugins would be great. To be able to link changes to changes in values would be double nice.

In addition to being able to quickly communicate value levels etc, being able to change background color would be great to fit in with other services where Grafana graphs are embedded. It would be nice to be able to blend in with a slightly different color scheme.

**Comment by oscarkilhed** [comment]
Some considerations:
Could potentially hide data points if background color matches series color.

interaction with light/dark/custom theme
